dessia_common/workflow/core.py

```diff
@@ -678,9 +725,38 @@ def dict_to_arguments(self, dict_: JsonSerializable, method: str):
             return arguments
         raise NotImplementedError(f"Method {method} not in Workflow allowed methods")
 
-    def method_dict(self, method_name: str = None, method_jsonschema: Any = None):
+    def _run_dict(self) -> Dict:
```

Comment:
We mighty have already talk about this, but I have hard time understanding why we need to copy the workflow, here

Reply:
In order to have the good input index in the dict, we must clear the imposed_variable_values. 
But, clearing the ivvs and then looping on inputs is still wrong because the inputs list is not up to date (it does not contain the inputs which were imposed).

So, by clearing the ivvs then copying the current workflow, I am sure that I have the good input indices